www/js/lib/zimfile.js

```diff
@@ -101,7 +102,8 @@ define(['xzdec_wrapper', 'zstddec_wrapper', 'util', 'utf8', 'zimDirEntry', 'file
      * @property {Integer} mimeListPos Position of the MIME type list (also header size)
      * @property {Integer} mainPage Main page or 0xffffffff if no main page
      * @property {Integer} layoutPage Layout page or 0xffffffffff if no layout page
-     * @property {Map} mimeTypes The ZIM file's MIME type table rendered as a Map (calculated entry)
+     * @property {String} type Extended property: either 'open' for OpenZIM file type, or 'zimit' for the warc2zim file type used by Zimit (set in zimArchive.js)
```

Comment:
Regarding the naming of this variable, `type` looks a bit too broad to me. And, technically, Zimit files also comply to OpenZIM spec.
Maybe we could simply add a boolean variable `isZimit` instead of `type` ? And set it to true or false in `setZimType`

Reply:
Thanks for the suggestion. My only doubt is whether there might be other types in the future that we might want to use this property for  -- but if so it would be easy enough to change. Regarding whether Zimit files comply with the OpenZIM specification, I would say that they are probably intended to comply, but that they don't for all the reasons listed in https://github.com/openzim/warc2zim/issues/86. However, on balance, they have enough resemblance to the OpenZIM spec to be a special case of that spec currently undocumented.

In any case, I'm happy to change this to a Boolean.

Reply:
OK no problem, you know the topic much better than I do.
You can keep it as it is if you whish

Reply:
I've changed the name of this property to `zimType`, which is more specific than `type`, but I've left it as a string in case we end up with some more types in the future.